src/main/webapp/search-interview.js

```diff
@@ -38,7 +38,25 @@ function selectInterview(interviewer) {
       `PM with a ${interviewer.getAttribute('data-company')} ` +
       `${interviewer.getAttribute('data-job')}. Check your email for more ` +
       `information.`);
-    // TODO: Call a servlet to save this selection.
+      
+    let requestObject = {
+    startTime: '2020-07-05T18:30:00Z',
+    endTime: '2020-07-05T19:30:00Z',
+    interviewer: interviewer.getAttribute('data-email'),
+    interviewee: 'user@gmail.com'
+    };
+    let requestBody = JSON.stringify(requestObject); 
+    let request = new Request('/scheduledInterview', {method: 'POST', body: requestBody});
+    
+    fetch(request)
+      .then(response => {
+        if (!response.ok) {
+          throw Error(`Status:${response.status}`); 
+        }
+      })
+      .catch((error) => {
+        console.log(error); 
```

Comment:
Maybe an alert would be more appropriate since all of our errors reflect some illegal action from the user.